packages/rest/src/rest.server.ts

```diff
@@ -1070,12 +1070,15 @@ const OPENAPI_SPEC_MAPPING: {[key: string]: OpenApiSpecForm} = {
 export interface OpenApiSpecOptions {
   /**
    * Mapping of urls to spec forms, by default:
-   * ```
+   * <br>
```

Comment:
Did you try to use `@remarks` followed by the code snippet?

Reply:
@raymondfeng I also tried using `@remarks` without the 3 backticks, and same result, the content disappears. Instead of showing up in another place.